src/test/java/com/fasterxml/uuid/impl/UUIDUtilTest.java

```diff
@@ -61,6 +61,16 @@ public void testExtractTimestampUUIDEpochBased() {
         }
     }
 
+    public void testExtractTimestampUUIDEpochRandomBased() {
+        TimeBasedEpochRandomGenerator generator = Generators.timeBasedEpochRandomGenerator();
+        final Random rnd = new Random(3);
```

Comment:
Ah ok so this `Random` is just for timestamp, but otherwise we get sort of arbitrary `SecureRandom`.
I guess that's about what we can verify.